From 5dcd12ec701b959aa1208cf2287c52baa0dbe739 Mon Sep 17 00:00:00 2001
From: srihars1 <sriharsha.govinda.raj@intel.com>
Date: Thu, 30 Apr 2015 16:07:02 +0530
Subject: [PATCH] Add missing pmic_chgdisctrl in pmic_regs_name

pmic_chgdisctrl was added in pmic_regs but is missing
in pmic_regs_name affecting the build of the debugfs
tree.

Change-Id: Iaed9a1549a7a456519e94a8410c891cab110182a
Tracked-On: https://jira01.devtools.intel.com/browse/AKP-1518
Signed-off-by: Bidipta Samadder <bidiptax.samadder@intel.com>
---
 drivers/power/intel_pmic_ccsm.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/power/intel_pmic_ccsm.h b/drivers/power/intel_pmic_ccsm.h
index fca3149beefc..1b594eea2fc1 100644
--- a/drivers/power/intel_pmic_ccsm.h
+++ b/drivers/power/intel_pmic_ccsm.h
@@ -318,6 +318,7 @@ char *pmic_regs_name[] = {
 	"pmic_mchgrirq1",
 	"pmic_chgrctrl0",
 	"pmic_chgrctrl1",
+	"pmic_chgdisctrl",
 	"pmic_lowbattdet0",
 	"pmic_lowbattdet1",
 	"pmic_battdetctrl",
-- 
1.9.1

